torch/quantization/__init__.py

```diff
@@ -15,13 +16,17 @@ def default_eval_fn(model, calib_data):
 
 _all__ = [
     'QuantWrapper', 'QuantStub', 'DeQuantStub', 'DEFAULT_MODULE_MAPPING',
-    # Top level API for quantizing a float model
+    # Top level API for eager mode quantization
     'quantize',
-    # Sub functions called by quantize
+    # Sub functions used by eager mode quantization
     'prepare', 'convert',
     # Sub functions for `prepare` and `swap_module`
     'propagate_qconfig', 'add_quant_dequant', 'add_observer', 'swap_module',
     'default_eval_fn',
+    # Top level API for graph mode quantization
+    'quantize_script',
```

Comment:
since graph mode is not yet ready for public light yet - can we remove it from __all__ here and maybe add underscore to `quantize_script` (this might be optional). We can codemod it back for 1.3